src/library/hooks/useLocation.js

```diff
@@ -0,0 +1,84 @@
+/* eslint-disable react-hooks/exhaustive-deps */
+import {useEffect, useState} from 'react';
+import {Dimensions} from 'react-native';
+import Geocoder from 'react-native-geocoder';
+import Geolocation from '@react-native-community/geolocation';
+
+Geocoder.fallbackToGoogle('AIzaSyB8WtEVYUasBc0RXOj60P2Y-HyuI3m_BjQ');
```

Comment:
Is this an API key? It is recommended to use an .env file to save the keys. Check this link:

[React Native dotenv](https://www.npmjs.com/package/react-native-dotenv)